app/services/onlinetesting/Phase3TestService.scala

```diff
@@ -66,7 +64,6 @@ object Phase3TestService extends Phase3TestService {
 }
 
 trait Phase3TestService extends OnlineTestService with Phase3TestConcern {
-  val adjustmentsService: AdjustmentsManagementService
```

Comment:
Seems like there are quite a few dependencies mixed in from OnlineTestService that don't need to be redeclared here
```
  val phase3TestRepo: Phase3TestRepository
  val launchpadGatewayClient: LaunchpadGatewayClient
  val tokenFactory: UUIDFactory
  val dateTimeFactory: DateTimeFactory
  val emailClient: OnlineTestEmailClient
  val auditService: AuditService
  val gatewayConfig: LaunchpadGatewayConfig
```
could go to
```
  val phase3TestRepo: Phase3TestRepository
  val launchpadGatewayClient: LaunchpadGatewayClient
  val gatewayConfig: LaunchpadGatewayConfig
```